convert2rhel/unit_tests/checks_test.py

```diff
@@ -695,39 +695,27 @@ def test_bad_kernel_substring(kernel_release, exp_return, monkeypatch):
 
 
 @pytest.mark.parametrize(
-    ("kernel_release", "kernel_pkg", "kernel_pkg_fingerprint", "exp_return"),
+    ("kernel_pkg_fingerprint", "exp_return"),
     (
         (
-            "4.18.0-240.22.1.el8_3.x86_64",
-            "kernel-core",
             "05b555b38483c65d",
             False,
         ),
         (
-            "4.18.0-240.22.1.el8_3.x86_64",
-            "kernel-core",
             "somebadsig",
             True,
         ),
     ),
 )
 @centos8
 def test_bad_kernel_fingerprint(
-    kernel_release,
-    kernel_pkg,
     kernel_pkg_fingerprint,
     exp_return,
     monkeypatch,
     pretend_os,
 ):
-    run_subprocess_mocked = mock.Mock(spec=run_subprocess, return_value=(kernel_pkg, ""))
+    kernel_release = run_subprocess(["uname", "-r"], print_output=False)[0].rstrip()
     get_pkg_fingerprint_mocked = mock.Mock(spec=get_pkg_fingerprint, return_value=kernel_pkg_fingerprint)
-    monkeypatch.setattr(checks, "run_subprocess", run_subprocess_mocked)
-    monkeypatch.setattr(
-        checks,
-        "get_installed_pkg_objects",
-        mock.Mock(return_value=[kernel_pkg]),
-    )
```

Comment:
By removing this the tests fail as it gets out of range
```
>       kernel_pkg_obj = get_installed_pkg_objects(kernel_pkg)[0]
E       IndexError: list index out of range
```

Reply:
thank you for bringing that to my attention, I have gotten that fixed!